cdk/lib/support-workers.ts

```diff
@@ -98,22 +95,16 @@ export class SupportWorkers extends GuStack {
         Fn.importValue(table)
       ),
     });
+    const parameterStorePolicy = new PolicyStatement({
```

Comment:
For the new TS lambdas I am going to store config in parameter store rather than S3 as the Scala lambdas do.